test/integration/simple_operations_test.lua

```diff
@@ -279,7 +342,7 @@ add('test_delete', function(g)
     -- insert tuple
     local result, err = g.cluster.main_server.net_box:eval([[
         local crud = require('crud')
-        return crud.insert('customers', {id = 33, name = 'Mayakovsky', age = 36})
+        return crud.insert_object('customers', {id = 33, name = 'Mayakovsky', age = 36})
```

Comment:
I see comment `--insert tuple`, but then you call `insert_object`...

Maybe `tuple/object` functions should be tested in a separated tests? `test_delete` and `test_delete_object`?

Reply:
Sounds reasonable

Reply:
Done